features/payment-flows-and-management/fraud-blocklist.md

```diff
@@ -33,6 +33,7 @@ In summary, a blocklist feature empowers merchants to proactively manage the sec
 ## How does Blocklist work at Hyperswitch?
 
 Currently we support blocking three types of resources i.e. card numbers (payment instrument), card bin, and extended card bin.
+But before using any of these features. We need to enable our Blocklist Guard.
```

Comment:
_A prerequisite to use this feature is to enable it using the /blocklist API as mentioned below._